substrate/frame/contracts/src/schedule.rs

```diff
@@ -396,10 +396,12 @@ impl Default for Limits {
 }
 
 impl<T: Config> Default for InstructionWeights<T> {
-	/// We price both `i64.const` and `drop` as `instr_i64const / 2`. The reason
-	/// for that is that we cannot benchmark either of them on its own.
+	/// We price all of `i64.add`, `local.get` and `local.set` as `instr_i64add / 2`. The reason
+	/// for that is that we cannot benchmark either of them on their own and division by 2 makes
+	/// for a decent compromise between the optimization capabilities of Wasmi (stack) and
+	/// Wasmi (register).
 	fn default() -> Self {
-		Self { base: cost_instr!(instr_i64const, 1), _phantom: PhantomData }
+		Self { base: cost_instr!(instr_i64add, 1), _phantom: PhantomData }
```

Comment:
Since cost_instr macro is only used here, since we use wami gas metering
we should simplify it to 
```rust
macro_rules! cost_instr {
	() => {
		cost_instr_no_params!(instr_i64add)
			.saturating_sub((cost_instr_no_params!(instr_i64add) / 2))
	};
}
```
or just inlined it here. That would make the comment above easier to understand as well

Reply:
I agree but the bench bot still reports a bug for usage of `instr_i64add` so I am reluctant to more changes until this is resolved. (i have no idea what is wrong atm)

Reply:
Or is this bug to be expected since we changed the set of benchmark test cases between `master` and this PR?

Reply:
> This is weird to me since we change an automatically generated file but code review recommended to change this to make the CI happy so I do.

It's only updated when you run the benchmark or ask the bot to do it with `bot bench substrate-pallet --pallet=pallet_contracts`. But I believe the code needs to compile first, I usually add a default impl in the WeightInfo trait to make it work